lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -50,6 +51,12 @@ def do_execute(machine, command, options={})
           end
         }
 
+        if options[:metainfo]
```

Comment:
The 'standard' output from [here](https://github.com/mitchellh/vagrant/blob/d93a2d6afc80bd8c7db907c58d539f13223332d9/templates/locales/en.yml#L1105-L1117):
```
        The following SSH command responded with a non-zero exit status.
        Vagrant assumes that this means the command failed!
        %{command}
        Stdout from the command:
        %{stdout}
        Stderr from the command:
        %{stderr}
```